docs/learning/display/index.md

```diff
@@ -0,0 +1,48 @@
+# The iTwin.js Display System
+
+The iTwin.js display system is responsible for producing and rendering graphics to enable visualization of the contents of an iTwin. Those contents can include iModels, reality models, IoT sensor data, map imagery, and other data sources. Applications can also provide their own graphics via [Decorators](../frontend/ViewDecorations.md) and [TiledGraphicsProvider](./TiledGraphicsProvider.md)s.
+
+The display system is divided into two subsystems:
+- The WebGL-based tile renderer running on the [frontend](../frontend/index.md), responsible for displaying the contents of [Viewport]($frontend)s; and
+- The native-code tile generator running on the [backend](../backend/index.md), responsible for [producing tiles](./TileFormat.md) from the contents of an iModel.
```

Comment:
No, the backend specifically produces tiles of iModel tile format; Tiles.md discusses all types of tiles from any source in any format.

Reply:
I think it's just unclear that the reference to the concept of tiles is about the the iModel-specific tile format. Maybe the sentence could be changed to make that clear? Like:

> responsible for **producing iModel tiles**

or

> responsible for producing tiles in the **iModel tile format**

Perhaps this is because I'm not familiar with tile formats in general. It's also not that bothersome, I can see it being left as is.

Reply:
See how  you like it now.